.changeset/shaggy-moons-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": minor
+---
+
+Add experimental support for the actions API. Actions let you define type-safe endpoints you can query from client components with progressive enhancement built-in.
```

Comment:
```suggestion
Adds experimental support for the Actions API. Actions let you define type-safe endpoints you can query from client components with progressive enhancement built in.
```
`Actionally`, we *do* capitalize API names, so let's give you a big ol' flashy capital A!

(And if we don't, we should, so consider it so!)

Reply:
Ah cool, wasn't sure if we did! Definitely think a capital will make Actions easier to call out in the docs

Reply:
Yes, as long as we are naming the "Actions API" then A is capitalized. When we're just saying "actions", it's not.